ryft-ui/src/main/java/com/ryftpay/android/ui/service/request/TransactionInfo.kt

```diff
@@ -0,0 +1,39 @@
+package com.ryftpay.android.ui.service.request
+
+import com.ryftpay.android.core.model.payment.PaymentSession
+import java.util.Currency
+
+internal data class TransactionInfo(
```

Comment:
is this GooglePay specific data?
Perhaps we place this under a googlepay package?

Reply:
yeah it is - updated 👍